manifests/server/yum.pp

```diff
@@ -33,5 +33,7 @@
     }
   }
 
-  package { 'createrepo': ensure => 'latest' }
+  package { 'createrepo': 
+    ensure => 'latest'
+  }
```

Comment:
Like above, this is style. I am apposed to this one. While placing class, resource, or function parameters on separate lines can aid in readability and code management, when there is only one parameter I prefer to keep the code tight for single-line readability.